tests/ops/test_simple_gla.py

```
@@ -318,6 +318,153 @@ def test_chunk_varlen(
+        scale=scale,
+        initial_state=h0,
+        output_final_state=True,
+    )
+    ((ref * do).sum() + (dht * ref_ht).sum()).backward()
+    ref_dq, q.grad = q.grad.clone(), None
+    ref_dk, k.grad = k.grad.clone(), None
+    ref_dv, v.grad = v.grad.clone(), None
+    ref_dg, g.grad = g.grad.clone(), None
+    ref_dh0, h0.grad = h0.grad.clone(), None
+
+    tri, tri_ht = fused_chunk_simple_gla(
```

Ruff (0.11.9)

Undefined name `fused_chunk_simple_gla`

(F821)